plugins/lookup/conjur_variable.py

```diff
@@ -97,6 +97,7 @@
 from stat import S_IRUSR, S_IWUSR
 from tempfile import gettempdir, NamedTemporaryFile
 import yaml
+import backoff
```

Comment:
This should be a try/except block as described in [https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html](https://docs.ansible.com/ansible/devel/dev_guide/testing/sanity/import.html)
```
try:
    import backoff
except ImportError as imp_exc:
    BACKOFF_IMPORT_ERROR = imp_exc
else:
    BACKOFF_IMPORT_ERROR = None
 ```

Reply:
I'll take a look. Thanks.

Reply:
@rpothier, Same result. It looks like the module isn't found, although it is supported officially.

Reply:
I'm looking into it also. When you say it's officially supported, do you mean by python? or Ansible?

Reply:
Hi @go-dima I did some research with Ansible and it looks like they do not support non standard libraries.
Ansible has `ansible.module_utils.api.retry` can you use that instead?

Reply:
Looks like ansible retry doesn't fit - it expects an exception to identify failure for retry. I'll keep looking for a suitable solution

Reply:
@rpothier 
I decided to implement it myself, care to tae a look? 
I also need an approval to run workflows now :)